tik/lib/include/tik/TikKernel.h

```diff
@@ -2,10 +2,15 @@
 
 namespace TraceAtlas::tik
 {
+    /// @brief Constructs a kernel function object from a tik file
+    ///
+    /// The constructor of this class will either construct a TikKernel if the given argument is actually a kernel function, or will return an invalid TikKernel
+    /// A function is only a valid TikKernel if it has Boundaries, Entrances, and Exits metadata attached to it.
+    /// If the function is valid, the constructor will allocate a new function with the context of the input arg.
     class TikKernel : public Kernel
     {
     public:
         TikKernel() = default;
-        TikKernel(llvm::Function *kernFunc);
+        TikKernel(llvm::Function *);
```

Comment:
Why did you remove this argument name?

Please avoid cosmetic changes like this if there is no reason. It makes the diffs harder to read.

Reply:
I changed these things because every time I change a function signature I have to do it twice. When theres a change to an arg name, I dont have to do it twice. Though I'll add these back anyway

Reply:
The arg names don't have to match when compiling. They should match when doing a pull request though.